### PR TITLE
Icon Selector for Kentico Cloud

### PR DESCRIPTION
**Pull request by emmanueltissera**
### Motivation

Which issue does this fix? N/A

If no issue exists, what is the fix or new feature? Were there any reasons to fix/implement things that are not obvious?
This seems to be the obvious choice to make icon selection an easy task for content editors.

### Checklist

- [x] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [x] Docs have been updated (if applicable)
- [x] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

If manual testing is required, what are the steps?
See README.md

### CODE REVIEW

File: IconSelector/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Icon Selector
+
+In Kentico Cloud, an icon selector can be specified by using the `Multiple Choice` content element. But this lacks in usablility as the content author has to guess what icon will be displayed when a particular choice is made. The icon selector hopes to make the life of the content editor easier with an easy to use dropdown of icons.
+
+The icon selector allows the developer to specify in the configuration what labels are displays for particular icons and what values are to saved in the content element.
+
+You can test it by configuring https://emmanueltissera.github.io/custom-element-samples/IconSelector/index.html url for your custom element.
```
Comment (Simply007) [inline IconSelector/README.md]: Could you set this (and other) urls to point to the target repository? 

```suggestion
You can test it by configuring https://kentico.github.io/custom-element-samples/IconSelector/index.html url for your custom element.
```

File: IconSelector/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+This outputs to:
+```text
+projectManager
+```
+
+## Configuration
+
+The main `controlLabel` parameter is used to show the text on the dropdown.
+
+Your have to configure `controlLabel` and `options` parameters. There should be at least more than one option for this to work.
```
Comment (Simply007) [inline IconSelector/README.md]:
```suggestion
You have to configure `controlLabel` and `options` parameters. There should be at least more than one option for this to work.
> Default value for the `controlLabel` is 'Please select'.
```

File: IconSelector/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+The icon selector allows the developer to specify in the configuration what labels are displays for particular icons and what values are to saved in the content element.
+
+You can test it by configuring https://emmanueltissera.github.io/custom-element-samples/IconSelector/index.html url for your custom element.
+
+You can provide an array of icons to be displayed with their source files, labels and values.
+
+![screenshot](https://lh3.googleusercontent.com/0zFxpQlplzKOVyNDkmRP4vtiQ1fOrlBMJzJToEYK18FpaJ9kCwuFg30CkkbDgRd6TWSlzVZeXfhzh4Fn71mDy1hjmGspP1OWkgLDzHwd9ObcYeWie7hUI2P7uHj5_wSbyYT8gXVyBA=w822-h353-no)
+
+This outputs to:
+```text
```
Comment (Simply007) [inline IconSelector/README.md]:
```suggestion
```plain
```

File: IconSelector/README.md (hunk 1 above)
Comment (emmanueltissera) [reply IconSelector/README.md]: 🤦‍♂️ Fixed,

Comment (emmanueltissera) [reply IconSelector/README.md]: Fixed

File: IconSelector/README.md (hunk 2 above)
Comment (emmanueltissera) [reply IconSelector/README.md]: 🤦‍♂️ Fixed,

File: IconSelector/README.md (hunk 3 above)
Comment (emmanueltissera) [reply IconSelector/README.md]: Fixed.

**Comment by Simply007** [comment]
Great @emmanueltissera!

Thank you for your contribution.

I will squash and merge this pull request and then I will link some minor updates.

**Comment by Simply007** [comment]
Adding commit link to the correction:
https://github.com/Kentico/custom-element-samples/commit/b794c635bdabaf2b69ecfc2d61d1ac3f1589d1d4

**Comment by emmanueltissera** [comment]
Thanks, @Simply007. Appreciate it.

**Pull request merged**